openapi/integrator/merchant/server.yaml

```diff
@@ -356,13 +356,13 @@ components:
           type: string
           pattern: '^[0-9]{4}$'
         amount:
-          $ref: '#/components/schemas/Amount'
+          $ref: '../components.yaml#/components/schemas/Amount'
         merchantName:
           $ref: '#/components/schemas/merchantName'
         merchantDisplayName:
           $ref: '#/components/schemas/merchantDisplayName'
         merchantReference:
-          $ref: '#/components/schemas/merchantReference'
+          $ref: '../components.yaml#/components/schemas/merchantReference'
```

Comment:
I don't get these changes. How could things work before?

Reply:
I explained this previously. I do not know how it could. 

> Not sure why it worked with an older version. The references are correct to me only with my changes, because they refer to the components.yml. Both amount and merchantReference are not present in the server.yaml itself and that’s how the references as they were before could have worked. (edited) 
> f.ex, all the other places that refer to Amount  are doing it in the same way I’ve done now.